crowbar_framework/public/docs/cisco_ucs.md

```diff
@@ -75,10 +75,19 @@ All troubleshooting should be done within the Cisco UCS Manager interface.
 
 The UCS Platform Emulator can be [downloaded from Cisco's
 website](http://developer.cisco.com/web/unifiedcomputing/ucsemulatordownload)
-after registering for a free account.  This barclamp has been tested
-with version 2.1(2aPE1) of the emulator.  However at the time of writing, 
-the emulator does not honour power commands, so do not expect this 
-functionality to work.
+after registering for a free account.
+
+Steps to start the UCS Platform Emulator using KVM with virtual cloud on cloud-admin network
+
+* Extract the downloaded .ova file
+  `tar xvf UCSPE_$Version.ova`
+  This results in 3 files: UCSPE_$Version-disk1.vmdk, UCSPE_$Version.ovf UCSPE_$Version.mf
+* Convert the VMDK image to qcow2 (Because image created by VMDK version 3 is ReadOnly)
+  `qemu-convert -O qcow2 UCSPE_Version-disk1.vmdk UCSPE_$Version-disk1.qcow2`
+* Create a virtual disk with at least 20GB
+* Start the VM using ucs_vm1.xml file after updating the correct Image paths.
+
+This barclamp has been tested with version 3.0(2cPE1) of the emulator.
 
 The Emulator was designed to run as a VMware virtual machine.
```

Comment:
It feels like this paragraph should be the second paragraph (so before your changes).